internal/command/format/diff.go

```diff
@@ -454,11 +456,22 @@ func (p *blockBodyDiffPrinter) writeNestedAttrDiff(
 	p.buf.WriteString(p.color.Color("[reset]"))
 	p.buf.WriteString(strings.Repeat(" ", nameLen-len(name)))
 
-	if old.HasMark(marks.Sensitive) || new.HasMark(marks.Sensitive) {
-		p.buf.WriteString(" = (sensitive value)")
+	// Then schema of the attribute itself can be marked sensitive, or the values assigned
+	sensitive := attrWithNestedS.Sensitive || old.HasMark(marks.Sensitive) || new.HasMark(marks.Sensitive)
+	if sensitive {
+		p.buf.WriteString(" = (sensitive")
+		if attrWithNestedS.Sensitive {
+			p.buf.WriteRune(')')
+		} else {
+			p.buf.WriteString(" value)")
```

Comment:
I don't think there was a conscious decision to use `(sensitive)` vs `(sensitive value)` in any situations -- I think we can try to make the output more consistent by always using the latter, since it is a "value" we are hiding, rather than describing something already being shown.

Reply:
I was trying to match the behavior of sensitive standard types, which show `-> null` when they are to be unset or destroyed. Is that a separate bug?